src/gl/directx11/vcTexture.cpp

```diff
@@ -112,41 +112,29 @@ udResult vcTexture_Create(vcTexture **ppTexture, uint32_t width, uint32_t height
     // Manually generate MaxMipLevels levels of mip maps
     if (hasMipmaps && !isRenderTarget)
     {
-      const void *pLastPixels = pPixels;
+      const void* pLastPixels = pPixels;
```

Comment:
Your VS settings are set to 'left' for the pointer which is inconsistent with our coding standard.